modules/hardware/allwinner/default.nix

```diff
@@ -94,14 +101,17 @@ in
     (mkIf cfg.allwinner-a64.enable {
       system.system = "aarch64-linux";
     })
-    (mkIf cfg.allwinner-h3.enable {
+    (mkIf (cfg.allwinner-h3.enable) {
```

Comment:
I guess this is an accidental needless change from initially adding `&& cfg.allwinner-s3.enable`?

Anyway, that's not needed.

```suggestion
    (mkIf cfg.allwinner-h3.enable {
```

Reply:
Excellent sleuthing.